include/Page/Base.php

```diff
@@ -259,6 +260,7 @@ public function enqueue_script() {
 	 */
 	public function enqueue_styles() {
 		wp_enqueue_style( 'bulk-move', $this->get_plugin_dir_url() . 'assets/css/bulk-move.css', array(), BM_BulkMove::VERSION );
+		wp_enqueue_style( 'select2', $this->get_plugin_dir_url() . 'assets/css/select2.min.css', array(), '4.0.0' );
```

Comment:
Add this enqueue before `bulk-move` and then add select2 as dependencies to bulk-move.

Reply:
@rajanit2000 

You have not added `select` as the dependency on both `bulk-move.js` and `bulk-move.css`